app/models/podcast.rb

```diff
@@ -27,6 +27,18 @@ class Podcast < BaseModel
   validates_associated :itunes_image, :feed_image
   validates :path, :prx_uri, :source_url, uniqueness: true, allow_nil: true
   validates :restrictions, media_restrictions: true
+  validates :explicit, inclusion: { in: %w(true false) }, allow_nil: false
+
+  # these keep changing - so just translate to the current accepted values
+  EXPLICIT_ALIASES = {
+    '' => 'false',
+    'no' => 'false',
+    'clean' => 'false',
+    false => 'false',
+    'yes' => 'true',
+    'explicit' => 'true',
+    true => 'true'
+  }.freeze
```

Comment:
To avoid test churn, and things like feed-imports that could have older data ... I'm just translating into the current true/false scheme.

But note that podcasts now have a _default_ of `"false"` - since it's required.  So if we're importing a feed with no channel-level explicit tag, we're going to mark it as clean in feeder.

Reply:
Yeah, this might need to be a) a checklist item for onboarding, or b) updated in the UI to prompt for a value rather than default on import?